acceptance/config/beaker/jenkins/redhat7-64m-64a.cfg

```diff
@@ -0,0 +1,30 @@
+---
+HOSTS:
+  redhat7-64-1:
+    roles:
+    - agent
+    - master
+    hypervisor: vcloud
+    pe_dir: 
+    pe_ver: 
+    pe_upgrade_dir: 
+    pe_upgrade_ver: 
+    platform: el-7-x86_64
+    template: Delivery/Quality Assurance/Templates/vCloud/redhat-7-x86_64
+  redhat7-64-2:
+    roles:
+    - agent
+    hypervisor: vcloud
+    pe_dir: 
+    pe_ver: 
+    pe_upgrade_dir: 
+    pe_upgrade_ver: 
+    platform: el-7-x86_64
+    template: Delivery/Quality Assurance/Templates/vCloud/redhat-7-x86_64
+CONFIG:
+  nfs_server: none
+  consoleport: 443
+  datastore: instance0
+  folder: Delivery/Quality Assurance/Enterprise/Dynamic
+  resourcepool: delivery/Quality Assurance/Enterprise/Dynamic
+  pooling_api: http://vcloud.delivery.puppetlabs.net/
```

Comment:
Same comment as above, this file was generated from version 0.12.1 of sqa-utils which includes the last for attributes.

Are you sure genconfig2 is handling this as expected?  Maybe there's a version mis-match?  I see the most recent tag as 0.12.1 in git, and when I install the gem from http://rubygems.delivery.puppetlabs.net I get version 0.12.1 which produces the output shown above.